src/features/content-explorer/content-explorer/__tests__/ContentExplorerFolderTreeBreadcrumbs.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import sinon from 'sinon';
```

Comment:
we've moved away from sinon. could we write new tests in enzyme or react testing-library?